### DISCUSSION TITLE
Renovate is still force pushing if branch is outdated

### DISCUSSION THREAD

**Discussion by xoxys** [renovatebot/renovate#8780] [Request Help] [answered]
Hi,

I have removed `:rebaseStalePrs` from my renovate preset but PR's getting still force pushed on every default branch change... See e.g. https://github.com/thegeeklab/docker-tidy/pull/44

The repo branch protection does also not enforce up to date branches:
![grafik](https://user-images.githubusercontent.com/3391958/108572214-1c84c180-7312-11eb-9fee-d89f8e86e8d4.png)

Any idea what I miss?

**Comment by rarkins** [comment, marked as answer]
Please see the documentation for "rebaseWhen". Also note that rebasing is mandatory if you enable automerge.

**Reply by xoxys** [reply]
> Also note that rebasing is mandatory if you enable automerge

That was the missing part. Thanks!
